src/components/MerchantsPage/styles.module.scss

```diff
@@ -152,6 +152,39 @@
   }
 }
 
+.bannerContentContainer {
```

Comment:
Is the css exactly the same as `contentContainer` except you have a `z-index: 1` and you added   `@include for-phone-only {
    height: 480px;`

Reply:
Yes 😬

Reply:
You are welcome to refactor my code at any time, happy to review the PR :)

Reply:
Is contentContainer even being used then...

Reply:
> You are welcome to refactor my code at any time, happy to review the PR :)

Yeah nah